app/serializers/webhook/event_serializer.rb

```diff
@@ -0,0 +1,10 @@
+module Webhook
+  class EventSerializer
+    include FastJsonapi::ObjectSerializer
+    set_type :webhook_event
+    set_id do |event|
+      "#{event.event_type}_#{event.timestamp}"
```

Comment:
I see two issues with the current approach: the prefix with the type means that by sorting by it all events are grouped by type which doesn't really help sequencing them. One could say: well, you can use the timestamp, yeah, but there's still the possibility that two events generated by two different Rails processes have the same timestamp thus having the same ID and we don't want that.

That's why distributed system usually rely on unique IDs for the concept of "event identifier". 

The easiest would be to slap a UUID there but we'd lose the sortability aspect of it. I'm going to think about it for a bit :D

Reply:
I should have commented it as a part of a self-review. Currently, `id` is filled up with a "placeholder" value, not the proper solution. Thanks for bringing this up. Finding an optimal solution is not easy, I'm open for a discussion. Maybe we could use a UUID for now.